flow/embedded_views.h

```diff
@@ -250,6 +250,14 @@ class ExternalViewEmbedder {
 
   virtual bool SubmitFrame(GrContext* context);
 
+  // Caller should make sure to call this after |SubmitFrame|.
```

Comment:
`This should only be called after |SubmitFrame|` reads better.

Reply:
done